### ISSUE TITLE
Question 'response to defenses of other organism involved in symbiotic interaction'

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#17930]
Is 'response to defenses of other organism involved in symbiotic interaction' really different from any 'response to defenses of other organism'? 

I think we should drop the 'involved in symbiotic interaction' from the term, as well as from the definition 'Any process that results in a change in state or activity of a cell or an organism (in terms of movement, secretion, enzyme production, gene expression, etc.) as a result of detecting the defenses of a second organism, ~where the two organisms are in a symbiotic interaction.~' 

@mgiglio99 @ValWood @addiehl 
What do you think ? 

Thanks, Pascale

**Comment by addiehl** [comment]
I am happy to drop the phrase as unnecessary for the term name and definition.
Alex

**Comment by mgiglio99** [comment]
Would it keep its 'symbiotic process' parent?

**Comment by pgaudet** [comment]
Symbiotic process is now defined as a process that is mediated symbiont genes, so both these terms ('response to defenses of other organism involved in symbiotic interaction' or 'response to defenses of other organism') would anyway NOT be under symbiotic process, but under 'interspecies interaction between organisms'. I hope that's OK. 

Pascale

**Comment by mgiglio99** [comment]
Whoa - I didn't notice that aspect of the def for symbiotic process.
I think symbiotic process has to include things that the host does.
We have lots of terms under there from the perspective of the host and those make sense and are needed. So I think that def should change. How about "A process carried out by gene products from an organism living in more or less intimate association with another organism that enables or is part of the interaction between the two organisms." then the rest as it is.
And I'm not following why 'response to defenses of other organism involved in symbiotic interaction' wouldn't be a child of 'symbiotic process'.

**Comment by ValWood** [comment]
@pgauget, maybe this can close?

**Comment by pgaudet** [comment]
The remaining issue is the definition for symbiotic process. 

With Paul we had changed the definition to try to separate what the host and what the symbiont does, but @mgiglio99 thought that symbiotic processes should include genes from both species - to be revisited. 

Pascale

**Comment by ValWood** [comment]
> @mgiglio99 thought that symbiotic processes should include genes from both species 

@mgiglio is that what you meant?

There are terms under symbiotic process that don't include genes from both species.

for example
GO:0075048    cell wall strengthening in symbiont involved in entry into host

 "interaction with host" is under "symbiotic process", but presumably not all "interaction with host" is symbiotic?

**Comment by mgiglio99** [comment]
Yes, symbiotic interaction needs to includes genes from both hosts and symbionts.
And yes I think all interaction with host is symbiotic. If you have a host, then that is a symbiotic relationship and any interactions between the organisms in that relationship would be a symbiotic process.

**Comment by mgiglio99** [comment]
Didn't we have another github issue where we were working on the def of symbiotic process?

**Comment by mgiglio99** [comment]
I found the other issue:
https://github.com/geneontology/go-ontology/issues/17979

**Comment by pgaudet** [comment]
Thanks @mgiglio99 . The present ticket is then redundant.

**Comment by mgiglio99** [comment]
Well, I'm not sure we ever resolved the original question.  We kind of got derailed due to the question of the def for 'symbiotic process'.
So, as it currently stands the original two terms have been merged - correct?
And it appears the merged term is not a child of 'symbiotic process' - correct?
It seems that it should be, or at least somehow related.  If the terms were merged because they meant the same thing, then the merged term should be a child of 'symbiotic process'. But I'm not sure all 'responses to defenses...' are the result of a symbiotic interaction.  I suppose that was the origin of the previous '...in symbiotic interaction' term.
